annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/WebAnnotationProtocolRest.java

```diff
@@ -1,199 +1,202 @@
+
+        // TODO userToken parameter has to be removed from HTTP request parameters		
+        // userToken = getUserToken(userToken, request);
+		
+	    verifyWriteAccess(WebAnnotationFields.CREATE_OPERATION, request);
+		
+		return storeAnnotation(wskey, null, provider, identifier, indexOnCreate, annotation, userToken);
+	}
+
+	@RequestMapping(value = "/annotations/", method = RequestMethod.POST, 
+			produces = { HttpHeaders.CONTENT_TYPE_JSONLD_UTF8, HttpHeaders.CONTENT_TYPE_JSON_UTF8})
+	@ApiOperation(notes = SwaggerConstants.SAMPLES_JSONLD, value = "Create annotations", nickname = "createAnnotations", response = java.lang.Void.class)
+	public ResponseEntity<String> createAnnotations(
+			@RequestParam(value = WebAnnotationFields.PARAM_WSKEY) String wskey,
+			@RequestParam(value = WebAnnotationFields.PROVIDER, required = false) String provider, 
+			@RequestBody String annotationPage,
+			@RequestParam(value = WebAnnotationFields.USER_TOKEN, required = false, defaultValue = WebAnnotationFields.USER_ANONYMOUNS) String userToken,
+			HttpServletRequest request)
+					throws HttpException, ApiKeyExtractionException, AuthorizationExtractionException {
+
+//		userToken = getUserToken(userToken, request);
+		
+//	    String jwtUserToken = verifyAccessRights(request, WebAnnotationFields.CREATE_OPERATION);
+		verifyWriteAccess(WebAnnotationFields.CREATE_OPERATION, request);
+		
+		
+		return storeAnnotations(wskey, provider, annotationPage, userToken);
+	}
+	
+	@RequestMapping(value = {"/annotation/{annoType}", "/annotation/{annoType}.jsonld"}, method = RequestMethod.POST, 
+			produces = { HttpHeaders.CONTENT_TYPE_JSONLD_UTF8, HttpHeaders.CONTENT_TYPE_JSON_UTF8})
+	@ApiOperation(notes = SwaggerConstants.SAMPLES_JSONLD, value = "Create annotation of given type", nickname = "createAnnotationByType", response = java.lang.Void.class)
+	public ResponseEntity<String> createAnnotationByTypeJsonld(@RequestParam(value = WebAnnotationFields.PARAM_WSKEY) String wskey,
+			@RequestParam(value = WebAnnotationFields.PROVIDER, required = false) String provider, 
+			@RequestParam(value = WebAnnotationFields.IDENTIFIER, required = false) String identifier,
+			@RequestParam(value = WebAnnotationFields.INDEX_ON_CREATE, required = false, defaultValue = "true") boolean indexOnCreate,
+			@RequestBody String annotation,
+			@RequestParam(value = WebAnnotationFields.USER_TOKEN, required = false, defaultValue = WebAnnotationFields.USER_ANONYMOUNS) String userToken,
+			@PathVariable(value = WebAnnotationFields.PATH_PARAM_ANNO_TYPE) String annoType,
+			HttpServletRequest request
+			)throws HttpException, ApiKeyExtractionException, AuthorizationExtractionException {
+
+//		userToken = getUserToken(userToken, request);
+//	    String jwtUserToken = verifyAccessRights(request, WebAnnotationFields.CREATE_OPERATION);
+		verifyWriteAccess(WebAnnotationFields.CREATE_OPERATION, request);
+		
+		MotivationTypes motivation = MotivationTypes.getTypeForOaType(annoType);
```

Comment:
this is wrong!  Please revert to the previous implementation. If there is a requirement to have "describe" available as own URL, than only the definition of the MotivationType needs to be updated.

Reply:
reverted